keyboards/nifty_numpad/keymaps/cadence/rules.mk

```diff
@@ -0,0 +1,6 @@
+TAP_DANCE_ENABLE=yes
+RGB_IDLE_ENABLE=yes
+
+ifeq ($(strip $(RGB_IDLE_ENABLE)), yes)
+    OPT_DEFS += -DRGB_IDLE_ENABLE
+endif
```

Comment:
Might make more sense to have this as a config.h define. Otherwise, this block should be moved to the keyboard's post_rules.mk
```suggestion
```

Reply:
I generally like the idea of controlling it as a config, but I think I prefer to keep it enabled/disabled at the keymap level. I've moved the lines to post_rules.mk and deleted them from rules.mk